.travis.yml

```diff
@@ -7,12 +7,12 @@ env:
   matrix:
     # Each line is a set of environment variables set before a build.
     # Thus each line represents a different build configuration.
-    - SHADERC_BUILD_TYPE=Release SHADERC_CODE_COVERAGE=OFF
```

Comment:
Did you really mean to remove those configurations?
The coverage generation is guarded by SHADERC_CODE_COVERAGE=ON anyway.